docs/rfcs/blocks-in-widget-areas.md

```diff
@@ -0,0 +1,535 @@
+# Blocks in Widget Areas RFC
+
+This RFC outlines the technical approach that we will take to upgrade the widget-editing areas in `wp-admin/widgets.php` and the Customizer to support blocks.
+
+Adding blocks to widget-editing areas was first announced at WCUS 2018, is a [Gutenberg Phase 2][phase-2] project, and appears in Matt's list of [9 Projects for 2019][2019-projects].
+
+![widgets](https://user-images.githubusercontent.com/1202812/51921660-d4ef0e00-23b5-11e9-86c9-5cb25c224d0b.gif)
+
+[phase-2]: https://github.com/WordPress/gutenberg/issues/13113
+[2019-projects]: https://make.wordpress.org/core/2018/12/08/9-priorities-for-2019/
+
+## Requirements
+
+- Editing blocks in `wp-admin/widgets.php` and the `wp-admin/customize.php` should use the same block editor that `wp-admin/post-new.php` uses.
+- The block editor should read and update blocks in widget-editing areas via the REST API.
+- Upgrading WordPress must not affect the appearance of the user's site, or any of their existing widgets.
+- Existing Core and third-party widgets must remain functional in the new block-based interface.
+- Backwards compatibility must be maintained. That is, themes and plugins that use public widget APIs must remain functional.
+
+## Terminology
+
+A _widget_ is an element in a WordPress site that displays some content. Widgets generally display the same content regardless of what post or page the user is viewing.
+
+A _widget area_ is an area in a WordPress site where widgets can be placed. The active theme defines how many widget areas there are, where they appear, and what they are called.
+
+Regular widgets can only be placed **once** into a widget area. A _multi widget_, however, can be placed several times into a widget area. When a multi widget is used several times, each usage is called an _instance_.
+
+Because many WordPress themes place widgets into a sidebar column, widget areas are often referred to as _sidebars_ or _dynamic sidebars_.
+
+**For more terminology and a detailed overview of how widgets _currently_ work in WordPress, see [#14182][14182].**
+
+[14182]: https://github.com/WordPress/gutenberg/issues/14182
+
+## Prior reading
+
+- [#14182][14182] is the issue which tracks adding an API for reading and updating blocks in widget areas.
+- [#13204][13204] contains design discussion about showing blocks in `wp-admin/widgets.php`.
+- [#13205][13205] contains design discussion about showing blocks in the Customizer.
+- [#13511][13511] is a PR which added support for rendering existing widgets as a block in Gutenberg.
+
+[13204]: https://github.com/WordPress/gutenberg/issues/13204
+[13205]: https://github.com/WordPress/gutenberg/issues/13205
+[13511]: https://github.com/WordPress/gutenberg/issues/13511
+
```

Comment:
I am going to add 
https://core.trac.wordpress.org/ticket/41683 
https://core.trac.wordpress.org/ticket/33507
https://core.trac.wordpress.org/ticket/35574
https://core.trac.wordpress.org/ticket/35669

Reply:
Thanks! Added in 19bf134271fbce739d453089fb7e9f24a61f9a28.